dataprofiler/data_readers/data.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import absolute_import
 from __future__ import division
-import logging
 
 from .csv_data import CSVData
 from .json_data import JSONData
 from .text_data import TextData
 from .parquet_data import ParquetData
 from .avro_data import AVROData
+from .. import dp_logging
 
-logger = logging.getLogger('DataProfiler.data')
+logger = dp_logging.get_logger().getChild(__name__.replace('dataprofiler.', ''))
```

Comment:
Maybe not in this PR if we don't have other changes to make, but would it beneficial to abstract this get child functionality where you only have to call: `dp_logging.get_child_logger(__name__)`?

Basically, makes it simple for future logging repeatability.

Reply:
Yeah thats a good idea, I'll add if other features get requested

Reply:
Added